yorkie/src/main/kotlin/dev/yorkie/api/CommonConverter.kt

```diff
@@ -5,9 +5,9 @@ import com.google.protobuf.ByteString
 import com.google.protobuf.kotlin.toByteString
 
 internal fun ByteString.toHexString(): String {
-    return BaseEncoding.base16().lowerCase().encode(toByteArray())
+    return BaseEncoding.base16().encode(toByteArray())
 }
 
 internal fun String.toDecodedByteString(): ByteString {
```

Comment:
I removed `lowercase()` since it seemed like to cause errors when capital letters are given as input.
but not sure it is a right solution.